packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -441,7 +441,7 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
         pointerEndId: _pointerQueue[1],
         pointerEndLocation: _pointerLocations[_pointerQueue[1]]!,
       );
-      _currentLine = null;
+      _currentLine = _initialLine;
```

Comment:
We need to initialize `_initialLine` and `_currentLine` at the same time when a new rotation coming, otherwise, they will be set to null in the next `_reconfigure` processing, which will cause this bug to occur.